snapshots/artificial_intelligence/2024-04-02/dynabench.xlsx.dvc

```diff
@@ -0,0 +1,39 @@
+# Learn more at:
+# http://docs.owid.io/projects/etl/architecture/metadata/reference/
+meta:
+  origin:
+    # Data product / Snapshot
+    title: "Dynabench: Rethinking Benchmarking in NLP"
+    description: |-
+      This dataset captures the progression of AI evaluation benchmarks, reflecting their adaptation to the rapid advancements in AI technology. The benchmarks cover a wide range of tasks, from language understanding to image processing, and are designed to test AI models' capabilities in various domains. The dataset includes performance metrics for each benchmark, providing insights into AI models' proficiency in different areas of machine learning research.
+
+      - MMLU (Massive Multitask Language Understanding): Tests AI's understanding across multiple disciplines, with metrics for knowledge comprehension and application.
```

Comment:
This hasn't formatted very nicely in my local set up. Does it look okay for you? 
<img width="576" alt="Screenshot 2024-04-02 at 15 15 19" src="https://github.com/owid/etl/assets/10774728/1a0690fd-afd4-4a7b-b921-46bc7078e30b">